basic_toolbox/css/scss/_module-toolbox.scss

```diff
@@ -23,6 +23,10 @@
 }
 
 .toolbox {
+  display: -webkit-box;
```

Comment:
@praxoc , please check your compiler to see if it has/support Autoprefix or not and if yes you don't need to add them again.